ui/src/setupTests.js

```diff
@@ -46,6 +46,8 @@ global.beforeEach(() => {
   const popupRoot = document.createElement('div');
   popupRoot.setAttribute('id', 'popup-root');
   document.body.appendChild(popupRoot);
+
+  window.localStorage.clear();
```

Comment:
I agree with your logic. Because this is in tests, it's fairly contained despite being a potential hazard as a global state change. If I were worried about it, I'd try to increase the visibility somehow. For example, have every test override a `WithLocalStorageCleared` base class. This would create some localized visibility for an effect that has wide implications.

Reply:
I think that's a good idea for when we do need this behavior, but until then I'm not worried about enabling this for all tests.